tests/integrationtest/t/planner/core/integration.test

```diff
@@ -172,15 +172,16 @@ create table strrange(a varchar(10) charset utf8mb4 collate utf8mb4_general_ci,
 insert into strrange values ('a', 1), ('A', 1), ('y', 1), ('Y', 1), ('q', 1);
 --sorted_result
 select * from strrange where a in ('a', 'y');
-create table strlist(a varchar(10) charset utf8mb4 collate utf8mb4_general_ci, b int) partition by list columns (a) (
-						partition p0 values in ('a', 'b'),
-						partition p1 values in ('c', 'd'),
-						partition p2 values in ('e', 'f'));
-insert into strlist values ('a', 1), ('A', 1), ('d', 1), ('D', 1), ('e', 1);
---sorted_result
-select * from strlist where a='a';
---sorted_result
-select * from strlist where a in ('D', 'e');
+# TODO: need to run with collation enabled.
```

Comment:
The orignal way is right. CI only check test cases with `new_collation_enable = false` when file name begins with `collation_`, like `collation_agg_func.test`

Reply:
Done